app/src/main/java/com/breadwallet/presenter/activities/intro/IntroActivity.java

```diff
@@ -158,10 +167,36 @@ public void run() {
                 APIClient apiClient = APIClient.getInstance(IntroActivity.this);
                 long endTime = System.currentTimeMillis();
                 Timber.d("timber: updateBundle DONE in %sms",endTime - startTime);
+
+
+                //DEV Moved this back until after the bundle is loaded
```

Comment:
This is WIP. Needs to be moved after debugging for Play Assets are present and tests veriify